File: JAVA PHASE GATE/SimpleArithmeticApp.java
import java.util.Scanner;
    public class SimpleArithmeticApp {
        public static void main(String...Odili) {
            Scanner input = new Scanner(System.in);

        
        int startTime = (int)System.currentTimeMillis();

            final int NUMBER_OF_QUESTIONS = 10;
            
            int countCorrect = 0;
            int answer = 0;
            int countNumberAnswered = 0;
            int countWrong = 0;
            int correctAnswer = 0;

        while(countNumberAnswered != NUMBER_OF_QUESTIONS){

        int firstNumber = (int)(Math.random() * 10);
        int secondNumber = (int)(Math.random() * 10);

         correctAnswer = firstNumber - secondNumber;

        if(firstNumber < secondNumber){
            int swap = firstNumber;
            secondNumber = swap;
            firstNumber = secondNumber;
        }

        
        System.out.print(firstNumber +  " - " + secondNumber + " = " );
             answer = input.nextInt();
            countNumberAnswered += 1;

     if(answer == correctAnswer){
           countCorrect += 1;                    
            }
       
        if(answer != correctAnswer){
            countWrong += 1;
           
        }
          
           
            if(countWrong >= 2) {
                break;
            }
    }

            int stopTime = (int)System.currentTimeMillis();
            int total = stopTime - startTime;

            System.out.println("The Number Of Correctly Answered Questions is " + countCorrect);

            System.out.println("Time taken is " + total/ 1000 + " seconds");
                

      




        }
    }

File: PYTHON PHASE GATE/Day 1/simple_arithmetic_app.py
import random

count_correct = 0
answer = 0
count_number_answered = 0
count_wrong = 0
correct_answer = 0
NUMBER_OF_QUESTIONS = 10           

while(count_number_answered != NUMBER_OF_QUESTIONS):
    first_number = random.randint(1,10)
    second_number = random.randint(1,10)

    correct_answer = first_number - second_number
    if(first_number < second_number):
        swap = first_number
        second_number = swap
        first_number = second_number
        print(first_number, " - " , second_number, " = ")
        answer = input("Answer : ")
        count_number_answered += 1

    if(answer == correct_number):
        count_correct += 1
    if(answer != correct_answer):
        count_wrong += 1



    print("The Number Of Correctly Answered Questions is " , countCorrect);
    print("Time taken is ", total/ 1000 ,  " seconds");
                            







